Update Aquality.Selenium.Core version

mialeska opened the pull request:
- Support overriding of capabilities from environment variables
- Support FindChildElements feature for Screen via ScreenElement property
- Add documentation to Screen methods. Make ElementFactory property static in to make it possible to be used during the field initialization

mialeska committed:
Update Aquality.Selenium.Core version
Support overriding of capabilities from environment variables
Support FindChildElements feature for Screen via ScreenElement property

paveliam reviewed (approved):
@mialeska please update template project too

mialeska committed:
Add documentation to screen methods.
Make ElementFactory property static in to make it possible to be used during the field initialization

mialeska committed:
Use GetValueDictionaryOrEmpty in DeviceSettings
